cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -232,17 +232,18 @@ class ReporterCommand : CliktCommand(
             }
         }
 
-        reportDurationMap.forEach { (entry, duration) ->
+        val successCount = reportDurationMap.count { (entry, duration) ->
             val (reporter, file) = entry
-            if (file.isFile) {
-                println(
+            file.isFile.also {
+                if (it) println(
```

Comment:
If the if spans multiple lines it should always use curly braces.

Reply:
Hmm, but it's not the `if` that spans multiple lines, it's just the `println` argument (like already before).

Reply:
(Actually, I admit I'm using this edge-case to "cheat" a bit as I don't want to wrap the string argument to `println`, which I would need to do if I added further indentation 😉)

Reply:
The Kotlin coding conventions have no opinion on that, so I'll accept the cheating ;)